web/client/components/TOC/fragments/template/index.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2017, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/*eslint-disable */
+
+const{Table} = require('react-bootstrap');
```

Comment:
Not just the Table component, all the bootstrap components should be in context

Reply:
Something like this?
```
const Components = require('react-bootstrap');

...
        "<Components.Table>" +
            "<thead>" +
            "<tr>" +
...
```

Reply:
Just rename Components to Bootstrap